lib/spark_api/authentication/oauth2_impl/faraday_middleware.rb

```diff
@@ -58,7 +58,7 @@ def on_complete(env)
               return
             end
           end
-          raise ClientError, {:message => "Unable to process sparkbar token #{body.inspect}", :code =>0, :status => env[:status]}
+          raise ClientError, {:message => "Unable to process sparkbar token #{body.inspect}", :code =>0, :status => env[:status], :request_path => env[:url]}
```

Comment:
Minimal change, but maybe we should deprecate this old class altogether https://github.com/sparkapi/spark_api/pull/114/files#diff-82ca4f3f5f54dc074483cf1d9b5a2da6L41